src/generator/operations.ts

```diff
@@ -395,6 +395,8 @@ function createProtocolRequest(client: string, op: Operation, imports: ImportMan
     for (const qp of values(aggregateParameters(op)).where((each: Parameter) => { return each.protocol.http !== undefined && each.protocol.http!.in === 'query'; })) {
       if (qp.required === true) {
         text += `\tquery.Set("${qp.language.go!.serializedName}", ${formatParamValue(qp, imports)})\n`;
+      } else if (qp.schema.type === SchemaType.Constant) {
+        // omit this query param. TODO once non-required constants are fixed
```

Comment:
Should we print something to the console so we know this isn't being implemented yet?

Reply:
Added